src/screens/SeasonsScreen.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+import { useLocation, Location, Link } from 'react-router';
+import { ShowData } from '../types';
+import { getTvDetails } from '../helpers';
+import { SeasonCard, SeasonCardLoader } from '../components';
+import { default as Typ } from '@mui/material/Typography';
+import { CardMedia } from '@mui/material';
+import ArrowBackRoundedIcon from '@mui/icons-material/ArrowBackRounded';
+
+/**
+ * Screen to render all of a TV Show's Seasons
+ */
+const SeasonsScreen: React.FC = (): JSX.Element => {
+    const location: Location = useLocation();
+    const showId = parseInt(location.pathname.split('/')[3]);
+    const [details, setDetails] = useState<ShowData>(
+        location.state ? location.state.details : null
+    );
+    const [loading, setLoading] = useState(details ? false : true);
+
+    useEffect(() => {
+        const handler = async () => {
+            const tvDetails = await getTvDetails(showId);
+            setDetails(tvDetails);
+            setLoading(false);
+        };
+        if (!details) handler();
+    }, [location]);
+
+    if (loading) {
+        return <SeasonCardLoader count={5} />;
+    }
+
+    return (
+        <section
+            data-testid='seasons-screen'
+            className='m-6 flex items-center sm:items-start flex-col w-[70svw] max-w-[1380px]'
+        >
+            <div className='mb-3 flex flex-col sm:flex-row items-center'>
+                <CardMedia
+                    component='img'
+                    className='rounded-sm'
+                    sx={{
+                        boxShadow: 5,
+                        width: 58,
+                        height: 87,
+                    }}
+                    image={
+                        details.poster_path
+                            ? `https://image.tmdb.org/t/p/w500${details.poster_path}`
```

Comment:
Use TMDB_BASE_PATH const

Reply:
The import would come from `'../__tests__/constants'`. 

I could be totally wrong, but I'm conflicted on the practicality of this inside actual components rather than test suites (where we're essentially just comparing values). Shouldn't our components explicitly state the image paths? 

If we did implement this constant in our components, our test suits would essentially just be `espect(TMDB_BASE_PATH).toBe(TMDB_BASE_PATH)`

@Thenlie

Reply:
> If we did implement this constant in our components, our test suits would essentially just be `expect(TMDB_BASE_PATH).toBe(TMDB_BASE_PATH)`

Yeah I think this is a good point.